src/zenml/models/v2/core/artifact_version.py

```diff
@@ -0,0 +1,392 @@
+        """
+        return self.get_body().type
+
+    @property
+    def artifact_store_id(self) -> Optional[UUID]:
+        """The `artifact_store_id` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().artifact_store_id
+
+    @property
+    def producer_step_run_id(self) -> Optional[UUID]:
+        """The `producer_step_run_id` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().producer_step_run_id
+
+    @property
+    def tags(self) -> List[TagResponseModel]:
+        """The `tags` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().tags
+
+    @property
+    def visualizations(
+        self,
+    ) -> Optional[List["ArtifactVisualizationResponse"]]:
+        """The `visualizations` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().visualizations
+
+    @property
+    def run_metadata(self) -> Dict[str, "RunMetadataResponse"]:
+        """The `metadata` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().run_metadata
+
+    @property
+    def materializer(self) -> Source:
+        """The `materializer` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().materializer
+
+    @property
+    def data_type(self) -> Source:
+        """The `data_type` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.get_metadata().data_type
+
+    # Helper methods
+    @property
+    def name(self) -> str:
+        """The `name` property.
+
+        Returns:
+            the value of the property.
+        """
+        return self.artifact.name
+
+    @property
+    def step(self) -> "StepRunResponse":
+        """Get the step that produced this artifact.
+
+        Returns:
+            The step that produced this artifact.
+        """
+        from zenml.artifacts.utils import get_producer_step_of_artifact
+
+        return get_producer_step_of_artifact(self)
+
+    @property
+    def run(self) -> "PipelineRunResponse":
+        """Get the pipeline run that produced this artifact.
+
+        Returns:
+            The pipeline run that produced this artifact.
+        """
+        from zenml.client import Client
+
+        return Client().get_pipeline_run(self.step.pipeline_run_id)
+
+    def load(self) -> Any:
+        """Materializes (loads) the data stored in this artifact.
+
+        Returns:
+            The materialized data.
+        """
+        from zenml.artifacts.utils import load_artifact_from_response
+
+        return load_artifact_from_response(self)
+
+    def read(self) -> Any:
+        """(Deprecated) Materializes (loads) the data stored in this artifact.
```

Comment:
Needs updating here? https://www.notion.so/zenml/Deprecations-903bcaacb5c04954b9f6133b3340a312

Reply:
Uhm yeah sure I'll add it (even though that deprecation is a year old by now 😅 )